modules/RTC/TraceablePeerConnection.js

```diff
@@ -268,6 +268,30 @@ export default function TraceablePeerConnection(
      */
     this.senderVideoMaxHeight = null;
 
+    // Set the codec preference that will be applied on the SDP
+    // based on the config.js settings.
+    if (this.options.preferH264 || this.options.preferVP9) {
```

Comment:
Are `preferH264` and `preferVP9` mutually exclusive? I would probably vote for a generic 'preferredCodec' option rather than adding `preferX`, `preferY`, `preferZ` and so on. The same applies for `disableH264` and `disableVP9` options.

Should we ever want to prefer or disable more than one codec then it would be easy to handle an array in future.

Reply:
Yes, `preferH264` and `preferVP9` are mutually exclusive. Bridge currently doesn't support H264 simulcast, so we would have to disable simulcast if we want to use H264 over jvb connection and our bridge doesn't support unicast VP9. VP9 has other issues like Safari doesn't support it yet and FF has a bug so we would prefer H264 over VP9 if both are present.
I left the `preferH264` option unchanged for backward compatibility.